src/components/Layout/LinesParams.tsx

```diff
@@ -68,8 +68,12 @@ export const LinesParams = ({ address, method, params }: LinesParamsProps) => {
   ) : decodedParams ? (
     <BaseTypeObjLines label='Params' data={decodedParams} />
   ) : params ? (
-    <Line label='Params'>
-      {params} ({decodeParamsError ? 'failed to decode' : 'decoding...'})
+    <Line
+      label={`Params (${
+        decodeParamsError ? 'failed to decode' : 'decoding...'
+      })`}
+    >
```

Comment:
Looks a bit nicer to have the info next to the label instead of the base64 data I think